web/src/components/pages/contribution/speak/speak.tsx

```diff
@@ -302,19 +302,25 @@ class SpeakPage extends React.Component<Props, State> {
   };
 
   private startRecording = async () => {
-    await this.audio.start();
-    this.maxVolume = 0;
-    this.recordingStartTime = Date.now();
-    this.recordingStopTime = 0;
-    this.setState({
-      // showSubmitSuccess: false,
-      recordingStatus: 'recording',
-      error: null,
-    });
+    try {
+      await this.audio.start();
+      this.maxVolume = 0;
+      this.recordingStartTime = Date.now();
+      this.recordingStopTime = 0;
+      this.setState({
+        // showSubmitSuccess: false,
+        recordingStatus: 'recording',
+        error: null,
+      });
+    } catch (err) {
+      this.setState({
+        recordingStatus: null,
+      });
+    }
   };
 
   private saveRecording = () => {
-    const RECORD_STOP_DELAY = 500;
+    const RECORD_STOP_DELAY = 300;
```

Comment:
could we revert this back to 500 or is it necessary? I think we had some reason why we wanted to go with that instead of a lower number.

Reply:
No, it not necessary. I forget to revert it.